### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by akrabat**
Run pip-compile and update all dependencies other than sphinx and reportlab.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,13 +2,13 @@
 # This file is autogenerated by pip-compile with Python 3.10
 # by the following command:
 #
-#    pip-compile --extra=aafiguresupport --extra=mathsupport --extra=plantumlsupport --extra=rawhtmlsupport --extra=sphinx --extra=svgsupport --extra=tests --output-file=requirements.txt setup.py
+#    pip-compile --extra=aafiguresupport --extra=mathsupport --extra=plantumlsupport --extra=rawhtmlsupport --extra=sphinx --extra=svgsupport --extra=tests --output-file=requirements1.txt setup.py
```
Comment (lornajane) [inline requirements.txt:5]: Why the changed filename?

**Comment by lornajane** [comment]
Note to self: reportlab upgrade is waiting for attention in #1110 because it changes some spacing/layout

**Comment by lornajane** [comment]
Why are we skipping sphinx this time? A non-scientific tests run with sphinx 6.1.3 and python 3.10 completed successfully for me

File: requirements.txt (hunk 1 above)
Comment (akrabat) [reply requirements.txt:5]: Because I failed to read the instructions properly when creating it...

Comment (akrabat) [reply requirements.txt]: > Why are we skipping sphinx this time? A non-scientific tests run with sphinx 6.1.3 and python 3.10 completed successfully for me

I saw it was a major version upgrade and assumed we'd break as we did with 4 to 5 without checking. As we don't, I've bumped this too.

**Comment by akrabat** [comment]
> Why are we skipping sphinx this time? A non-scientific tests run with sphinx 6.1.3 and python 3.10 completed successfully for me

I saw it was a major version upgrade and assumed we'd break as we did with 4 to 5 without checking. As we don't, I've bumped this too.

**Comment by akrabat** [comment]
Rebased against #1134 and picked up a couple of other updated dependencies.

**Comment by akrabat** [comment]
Rebased now that #1134 is merged.

**Review by lornajane — APPROVED** [review]
Thanks for bringing us up to date!

**Pull request merged**